src/nav/nav.jsx

```diff
@@ -10,17 +14,67 @@ const linksWordsList = linksWord.map(function (link) {
         </Link>
     );
 });
-const linksPhoto = ['person.svg', 'magnifier.svg', 'heart.svg', 'shoppingCart.svg']
-const linksPhotoList = linksPhoto.map(function (link) {
-    return (
-        <Link to=''>
-            <img
-                src={`/photos/${link}`}
-                alt={link} />
-        </Link>
-    );
-});
+const navLinks = [
+    {
+        path: 'person',
+        extraClass: false
+    },
+    {
+        path: 'magnifier',
+        extraClass: false
+    },
+    {
+        path: 'heart',
+        extraClass: false
+    },
+    {
+        path: 'shoppingCart',
+        extraClass: true
+    },
+]
+
+
+
+const LinksPhotoList = ({ links, handleOpenModal }) => {
+    return links.map(link => {
+        return (
+            <button onClick={link.extraClass ? handleOpenModal : () => { }} className={link.extraClass ? 'navig_item shopping_cart' : 'navig_item'}>
```

Comment:
very poor naming again. so `extraClass` property decides whether an even listener will be attached to this button, it's not transparent at all.
I would do it differently. so since your `shoppingCart` has such a different behavior you can use loop for the first 3 elements from `navLinks` array (basically remove `shoppingCart` from that array) and then render `shoppingCart` as a standalone item